testplan/web_ui/testing/src/Nav/NavEntry.js

```diff
@@ -37,7 +37,13 @@ const NavEntry = (props) => {
   return (
     <div
       className='d-flex justify-content-between align-items-center'
-      style={{ height: "1.5em" }}
+      style={{
+        height: "1.5em",
+        webkitUserSelect: "text",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#browser_compatibility 
UserSelect is enough, all of the modern web browsers support user-select